src/light_data/mod.rs

```diff
@@ -103,6 +103,11 @@ impl LightData {
     pub fn to_trimmed_le_bytes(x: usize) -> Vec<u8> {
         x.to_le_bytes()[..Self::byte_count(x) as usize].to_vec()
     }
+    /// Returns the minimum number of bytes required to represent the integer.
+    pub fn trimmed_le_bytes_len(xs: &[u8]) -> usize {
+        // usize::from_le_bytes(xs.try_into().unwrap()) // this seems to break when loading big stores: TryFromSliceError(())
```

Comment:
That's the point of my remark: 
1. I'm not a big fan of testing in release mode,
2. [This issue should be more widely known](https://stackoverflow.com/questions/71196238/why-does-repeated-multiplication-panic-due-to-overflow-in-debug-mode-when-it-ou)